bigtable/snippets/writes/write_batch.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+# Copyright 2019, Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# [START bigtable_writes_batch]
+import datetime
+
+from google.cloud import bigtable
+
+
+def write_batch(project_id, instance_id, table_id):
+    client = bigtable.Client(project=project_id, admin=True)
+    instance = client.instance(instance_id)
+    table = instance.table(table_id)
+
+    timestamp = datetime.datetime.utcnow()
```

Comment:
Question: Any gotchas to know about timestamps? What happens if I use a timestamp in the future or past?

Hopefully this is covered in prose around the samples if it's too much for a comment.

Reply:
any timestamps are okay to use